Gemfile.lock

```diff
@@ -25,6 +25,7 @@ PATH
       jwt (>= 2.1.0, < 3)
       mini_magick (>= 4.9.4, < 5.0.0)
       multipart-post (~> 2.0.0)
+      naturally (~> 2.2)
```

Comment:
Apparently `simctl` depends on it🙂
https://github.com/fastlane/fastlane/blob/0d1aa50045d57975d8b9e5d5f1f489d82ee0f437/Gemfile.lock#L293-L295